client/package.json

```diff
@@ -16,13 +16,18 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "babel-loader": "^8.1.0",
+    "dotenv": "^8.2.0",
```

Comment:
I don't think you need to install `dotenv` on the client. It is already bundled in your React app. See https://create-react-app.dev/docs/adding-custom-environment-variables

Reply:
Thanks!